services/s3/src/test/java/software/amazon/awssdk/services/s3/internal/multipart/MultipartDownloaderSubscriberWiremockTest.java

```diff
@@ -98,32 +98,6 @@ <T> void happyPath_shouldReceiveAllBodyPartInCorrectOrder(AsyncResponseTransform
         util.verifyCorrectAmountOfRequestsMade(amountOfPartToTest);
     }
 
-    @ParameterizedTest
-    @MethodSource("argumentsProvider")
-    <T> void errorOnFirstRequest_shouldCompleteExceptionally(AsyncResponseTransformerTestSupplier<T> supplier,
```

Comment:
Removing this test as it now hangs, since logic is changed to depend on DownloadObjectHelper to propagate the exception for the first part, rather solely in the MultipartDownloaderSubscriber

- https://github.com/aws/aws-sdk-java-v2/blob/0381604ca45769589734c9c638806ab34df7e8a5/services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/multipart/DownloadObjectHelper.java#L53

Reply:
Did we add similar tests to verify this behavior?

Reply:
Should be covered by
https://github.com/aws/aws-sdk-java-v2/blob/54457df9020890a42456a3cbdb55586705cb8821/services/s3/src/test/java/software/amazon/awssdk/services/s3/internal/multipart/S3MultipartClientGetObjectWiremockTest.java#L238